### PR TITLE
enroll-keys: implement --export

### PR DESCRIPTION
**Pull request by Foxboron**
Export the keys we intend to enroll as .auth or .esl files

### CODE REVIEW

**Comment by Foxboron** [comment]
cc @RaitoBezarius

File: docs/sbctl.8.txt [hunk 1]
```diff
@@ -67,6 +67,12 @@ EFI signing commands
                 files and unset the immutable attribute before enrolling
                 certificates.
 
+        *--export*;;
+                Export the keys we intend to enroll as EFI Signatlure Lists
```
Comment (RaitoBezarius) [inline docs/sbctl.8.txt:71]:
```suggestion
                Export the keys we intend to enroll as EFI Signature Lists
```

**Review by RaitoBezarius — COMMENTED** [review]
Thank you so much for getting to it! If you want, I will drive it in Lanzaboote for auto secureboot enroll via sd-boot. :)

File: docs/sbctl.8.txt (hunk 1 above)
Comment (Foxboron) [reply docs/sbctl.8.txt:71]: Yes, also missing a note it's exporting it into the current working directory.

**Pull request merged**